src/main/java/com/kodilla/ecommercee/domain/ProductDto.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProductDto {
+    private String title;
```

Comment:
Czy dto jest potrzebne jeśli zawiera tylko jedną wartość?

Reply:
zaraz to zmienimy :D

Reply:
fixed